mobile_app_connector/mappings/compassion_donation_mapping.py

```diff
@@ -19,7 +19,15 @@ class MobileDonationMapping(OnrampMapping):
     CONNECT_MAPPING = {
         "DisplayOrder": "id",
         "FundName": "name",
-        "Id": "id"
+        "Id": "id",
+        "ImageIcon": "image_icon",
     }
 
     FIELDS_TO_SUBMIT = {k: None for k, v in CONNECT_MAPPING.iteritems() if v}
+
+    def _process_connect_data(self, connect_data):
```

Comment:
Is this function really doing something? I feel it is useless and only putting back the same values as the original ones...

Reply:
it was a test. sorry. I remove it.